src/mapping_navigation/scripts/navigation.py

```diff
@@ -34,7 +34,7 @@ def listener(self):
         rospy.init_node('navigation', anonymous=True)
         rospy.Subscriber('airsimPose', PoseStamped, self.handle_gps_data)
         rospy.Subscriber("sensor/speed", Float64, self.handle_speed_data)
-        rate = rospy.Rate(30)
+        rate = rospy.Rate(10)
```

Comment:
If the rate of navigation / pure pursuit is higher or equal to lidar, lidar obstacle avoidance suffers